### PR TITLE
[DOCS] Add GX Cloud Content

### PR DESCRIPTION
**Pull request by kwcanuck**
This PR adds content to the existing GX Cloud help file.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] _Appropriate_ tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Batch Requests specify a subset of data to select, as opposed to defining the data itself, so I would change `that define the data` to something like `to select the data`.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md:11]: I don't know where this doc snippet lives. Is it specific for cloud and snowflake? This prerequisites seem to indicate this. One can create in OSS alone and for datasources beside snowflake.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Do we need to indicate what GX uses temp tables for?

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Do we need to define what/how/when gx would test the Data Asset connection?

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: This name can be arbitrary but must be unique for assets related to this datasource. Do we want to be explicit about these requirements?

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md:66]: Do we need an introductory statement saying editing can't be done via the web ui currently? Jumping straight from a UI based workflow to a notebook one seems a little abrupt.

**Review by billdirks — COMMENTED** [review]
Thanks for doing this. I've only looked at the first file so far. I'll look at the others after my batch of meetings.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md:66]: @billdirks I am documenting how users can complete tasks **_currently_**. If they need to jump to a Jupyter Notebook, that's what they are instructed to do. This is the approach that Hernan and others requested. As you discovered, the gaps in functionality are obvious when the process is documented. Also, if we're allowing access to GX Cloud and the documentation, we can't leave content out because the functionality isn't available in the UI yet. Users need to be able to complete the necessary tasks.

**Review by billdirks — COMMENTED** [review]
I've looked at the other files now and they look good. My only comment there, is when I read docs, I often like to have screenshots for when I need to click on a button or tab that might be hard to find.

File: docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md
Comment (billdirks) [inline docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md]: Looks like there is an extra comma here.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (billdirks) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md:66]: I meant I thinkwe should be explicit that this functionality doesn't not that we should leave this out. That is, have an intro sentence saying: "We do not currently support editing expectations in the UI." before jumping into the jupyter notebook steps.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md:11]: Discussed with @billdirks and this can stay as is.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Temp table = To store information temporarily and improve performance by making queries run faster.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Test to make sure the configuration is current to avoid errors and reduce downtime. When you define the configuration the connection is tested. Preventative.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: Data Source and Data Asset names must be unique. A Data Asset with the same name can be used but must be associated with unique Data Sources.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]:
```suggestion
A Data Asset is a collection of records that you create when you connect to your Data Source. When you connect to your Data Source, you define a minimum of one Data Asset. You use these Data Assets to create the Batch Requests that select the data that is provided to your Expectations.
```

**Review by Kilo59 — APPROVED** [review]
The manage_users section seems fine to me.

I would suggest smaller PRs in the future.

Please address @billdirks concerns before merging.

**Review by billdirks — APPROVED** [review]
@kwcanuck and I talked through my comments earlier today.

File: docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/checkpoints/manage_checkpoints.md]: @billdirks - good catch! This has been corrected.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @billdirks I have added the text as discussed. Thank you.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @billdirks I have added the text as discussed. Thank you.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @billdirks I have added the text as discussed. Thank you.

**Review by joshzzheng — APPROVED** [review]
It'd be great if we can set a max width for the Snowflake box. Shouldn't block the release though.

**Pull request merged**
